resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -1,115 +1,370 @@
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeAndModifyInstanceAttribute",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeInstanceAttribute",
+        "ec2:ModifyInstanceAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeTransitGateway",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeTransitGatewayAttachments",
+        "ec2:DescribeTransitGatewayConnectPeers",
+        "ec2:DescribeTransitGatewayConnects",
+        "ec2:DescribeTransitGatewayMulticastDomains",
+        "ec2:DescribeTransitGatewayPeeringAttachments",
+        "ec2:DescribeTransitGatewayRouteTables",
+        "ec2:DescribeTransitGateways",
+        "ec2:DescribeTransitGatewayVpcAttachments",
+        "ec2:GetTransitGatewayAttachmentPropagations",
+        "ec2:GetTransitGatewayPrefixListReferences",
+        "ec2:GetTransitGatewayRouteTableAssociations",
+        "ec2:GetTransitGatewayRouteTablePropagations"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "SearchTransitGatewayRoutes",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:SearchTransitGatewayRoutes"
+      ],
+      "Resource": "arn:aws:ec2:*:*:transit-gateway-route-table/*"
+    },
+    {
+      "Sid": "DescribeVpc",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeVpcClassicLink",
+        "ec2:DescribeVpcClassicLinkDnsSupport",
+        "ec2:DescribeVpcEndpointConnectionNotifications",
+        "ec2:DescribeVpcEndpointServiceConfigurations",
+        "ec2:DescribeVpcEndpointServices",
+        "ec2:DescribeVpcPeeringConnections"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeVpcAttribute",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeVpcAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:vpc/*"
+    },
+    {
+      "Sid": "GetS3objectMetadata",
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObjectAcl",
+        "s3:GetObjectTagging"
+      ],
+      "Resource": [
+        "arn:aws:s3:::*-image-registry-${aws:RequestedRegion}-*"
+      ],
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "GetAndCreateTirosQueries",
+      "Effect": "Allow",
+      "Action": [
+        "tiros:CreateQuery",
+        "tiros:GetQueryAnswer",
+        "tiros:GetQueryExplanation"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "CreateRedHatManagedTag",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateTags"
+      ],
+      "Resource": [
+        "arn:aws:ec2:*:*:snapshot/*",
+        "arn:aws:ec2:*:*:volume/*",
+        "arn:aws:ec2:*:*:network-insights-path/*"
```

Comment:
Please separate this into a different block so we have one for the verifier related actions and one for the network insights path

Reply:
yes the block for NetworkVerifier is on line 299

Reply:
Ah this is for Snapshot and NetworkInsightsPath. Can we separate those as well?: